### PR TITLE
chore: migrate data from vue-hyderabad meetup

### PR DESCRIPTION
**Pull request by znck**

### CODE REVIEW

**Comment by znck** [comment]
`CODEOWNERS` works nicely in monorepo 👍

File: meetups/hyderabad/02-jingle-bells/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    sponsor: FactSet <@FactSet>
+organizers:
+  - Praveen Puglia <@praveenpuglia>
+  - Aravind Balla <@aravindballa>
+  - Krishna Nandula <@knandula>
+---
+
+<EventPage />
+## Google Photos Album 
+<!-- [https://photos.app.goo.gl/BU6QXzECrQBTG2dB9](https://photos.app.goo.gl/BU6QXzECrQBTG2dB9) -->
```
Comment (praveenpuglia) [inline meetups/hyderabad/02-jingle-bells/README.md]:
```suggestion
[https://photos.app.goo.gl/BU6QXzECrQBTG2dB9](https://photos.app.goo.gl/BU6QXzECrQBTG2dB9)
```

File: meetups/hyderabad/02-jingle-bells/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  map:
+    embed: true
+    embedCode: <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3806.785702828827!2d78.37239691493326!3d17.422068906389494!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x3bcb93f8722f8c33%3A0x6cd17cd26a48e99e!2sFactSet%20Systems%20India%20Private%20Limited%2C%20DivyaSree%20Orion!5e0!3m2!1sen!2sin!4v1575357472280!5m2!1sen!2sin" width="100%" height="450" frameborder="0" style="border:0;" allowfullscreen=""></iframe>
+agenda:
+  - title: The Art of Crafting Codemods
+    type: talk
+    speaker: Rajasegar Chandran <@rajasegar_c>
+    description: Codemod is a mechanism to make sweeping changes across your code with ease and effectiveness, assisting in large-scale migrations of the code-base. This can be performed through automated tools such as jscodeshift.
+    bio: Rajasegar works with Freshworks as a front-end developer. He is passionate about open-source software and currently writes codemods for the Ember community.
+    # deck: https://github.com/vuehyd/hub/blob/master/01-assemble/decks/assemble-talk-1.pdf
```
Comment (praveenpuglia) [inline meetups/hyderabad/02-jingle-bells/README.md]:
```suggestion
    deck: https://slides.com/rajasegarchandiran/codemods-vue
```

File: meetups/hyderabad/02-jingle-bells/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  - title: Introduction to Web Components
+    type: talk
+    speaker: Nagaraj Hubli <@nagaraj_hubli>
+    description: I will be covering the importance of Web Components, and how they are natural evolution of the Web, along with a brief case study of how it helped bootstrap applications in a large enterprise.
+    bio: I work at Pramati Technologies as a front end web developer, and spends all my free time studying about Bitcoin.
+    # deck: https://www.canva.com/design/DADqrYoCIt4/XtSLABwMQSp_5PdECnZwZQ/view
```
Comment (praveenpuglia) [inline meetups/hyderabad/02-jingle-bells/README.md]:
```suggestion
    deck: https://www.canva.com/design/DADt4vLEo0Y/oCECV6Oxi3rIlhJi5BNgpQ/view
```

File: meetups/hyderabad/02-jingle-bells/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  - title: Vuex ORM
+    type: talk
+    speaker: Ayush Akhoury
+    description: I will be covering VueX ORM plugin. It  is a plugin for Vuex to enable Object-Relational Mapping access to the Vuex Store. It helps to deal with nested/relational data on the front-end by decoupling them into separate modules.
+    bio: Frontend Engineer at FactSet
+    # deck: https://www.canva.com/design/DADqrYoCIt4/XtSLABwMQSp_5PdECnZwZQ/view
```
Comment (praveenpuglia) [inline meetups/hyderabad/02-jingle-bells/README.md]:
```suggestion
    deck: https://pro.ispringcloud.com/acc/wrV9x3UzNzkzNA/s/37934-opfNv-7391S-sB5ws
```

File: meetups/hyderabad/02-jingle-bells/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+sponsors:
+  - type: T-Shirt
+    sponsor: Voicezen<@voicezen>
+  - type: Venue and Snacks
```
Comment (praveenpuglia) [inline meetups/hyderabad/02-jingle-bells/README.md]:
```suggestion
  - type: Venue, Snacks, T-Shirt
```

**Review by praveenpuglia — CHANGES_REQUESTED** [review]

**Comment by praveenpuglia** [comment]
Looks good to me! @znck - LGTM!

**Comment by praveenpuglia** [comment]
Merged! 😝

**Pull request merged**